tests/sentry/issues/test_escalating.py

```diff
@@ -33,27 +33,28 @@ def setUp(self) -> None:
     def _load_event_for_group(
         self,
         project_id: Optional[int] = None,
-        minutes_ago: int = 1,
+        count: int = 1,
+        hours_ago: int = 0,
```

Comment:
Using hours_ago lines up more closely with the hourly bucket we use in my Snuba query.